app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/utils.ts

```
@@ -0,0 +1,40 @@
+  if (typeof a === "number" && typeof b === "number") {
+    if (percentage) {
+      return formatValue(((a - b) / a) * 100, fractionDigits);
+    }
+    return formatValue(a - b, fractionDigits);
+  }
+}
+
+export function formatValue(value: string | number, fractionDigits = 3) {
+  const numericValue =
+    typeof value === "number" ? value : parseFloat(value as string);
+  if (!isNaN(numericValue)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)